abipy/lessons/lesson_helper_functions.py

```diff
@@ -8,10 +8,34 @@
 import sys
 import os
 import shutil
+import yaml
+import html2text
+from variables import *
 import abipy.abilab as abilab
 import abipy.data as abidata
 
 
+class VariableDatabase(object):
```

Comment:
https://github.com/gmatteo/abipy/commit/1778c7faecc1892c8ccfb8da4ae8f6eff1cb2b95

On Wed, Jan 21, 2015 at 10:08 PM, yanikou19 <notifications@github.com>
wrote:

> In abipy/lessons/lesson_helper_functions.py
> <https://github.com/gmatteo/abipy/pull/53#discussion_r23334452>:
>
> >  import abipy.abilab as abilab
> >  import abipy.data as abidata
> >
> >
> > +class VariableDatabase(object):
>
> Are you sure this global variable _VAR_DATABASE is correctly propagated to
> the other lessons if I import the get_database function in these lessons ?
> because it didn't work when I tried to save it as a global variable...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gmatteo/abipy/pull/53/files#r23334452>.
>



-- 
Nel 1969, con la potenza di calcolo di due Commodore 64, l'uomo è riuscito
a sbarcare sulla luna. Oggi, per far girare decentemente Windows Vista,
serve 1GB di RAM... qualcosa deve essere andato storto